src/devices/Ht16k33/Font.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.Ht16k33
+{
+    internal static class Font
+    {
+        /// <summary>
+        /// Used to mask upper 4 bits of a byte for a single hexadecimal value
+        /// </summary>
+        private const byte HexadecimalMask = 0b0000_1111;
+
+        /// <summary>
+        /// Hexadecimal digit (0..15) fonts
+        /// </summary>
+        private static readonly byte[] s_hexDigitFonts = new byte[]
+        {
+            0b0011_1111, // 0
```

Comment:
could these be defined as another enum values, i.e.:

```csharp
enum Segment : ...
{
...
LetterA = LeftBottom | LeftTop | ...,
}
```

similarly as we done in https://github.com/dotnet/iot/blob/master/src/devices/Tm1637/Character.cs

There is also a high chance you can re-use or copy&paste it (or at least most of it)

Reply:
I saw the similar enum in the other device project, but there are slight differences (e.g. dot handling), even though they are very close I wasn't quite sure how to reuse them in both places just yet.

I generated these "fonts" in Excel, I think it's pretty compact this way and it's not public anyway. If you insist I can convert them to enum flag combinations.

Reply:
@gLes I'm expecting as we add more segment displays in the future that we will eventually figure out how to unify this code a bit more so I'd prefer if the interface was as identical as possible.

I expect you will only need to change names to be the same and then copy & paste remaining definition from the other projects (minus not supported values)

Reply:
Should I move these to the Common folder then?

Reply:
Didn't make them a common item yet, but converted the binary values to enum combinations.